community/common/src/main/java/org/neo4j/kernel/api/exceptions/Status.java

```diff
@@ -422,6 +422,18 @@ public Code code()
                 "A malformed schema rule was encountered. Please contact your support representative." ),
         UnknownError( DatabaseError,
                 "An unknown error occurred." ),
+        OutOfMemoryError( TransientError,
+                "There is not enough memory to perform the current task. Please try increasing " +
+                "'dbms.memory.heap.max_size' in 'conf/neo4j-wrapper.conf' or if you are running an embedded " +
```

Comment:
Can we say something like 

     in the process wrapper configuration (normally in 'conf/neo4j-wrapper.conf' or, if you are using Neo4j Desktop,
     found through the user interface)

Since neo4j-wrapper.conf may be in different places, and doesn't exist in neo4j desktop.